src/AutoRest.CSharp/Common/AutoRest/Plugins/GeneratedCodeWorkspace.cs

```diff
@@ -24,6 +25,25 @@ internal class GeneratedCodeWorkspace
         public static readonly string SharedFolder = "shared";
         public static readonly string GeneratedFolder = "Generated";
 
+        private static readonly IReadOnlyList<MetadataReference> AssemblyMetadataReferences;
+
+        static GeneratedCodeWorkspace()
+        {
+            var references = new List<MetadataReference>
+            {
+                MetadataReference.CreateFromFile(typeof(object).Assembly.Location),
+                MetadataReference.CreateFromFile(typeof(Response).Assembly.Location),
```

Comment:
So you choose an arbitrary type from `azure.core`, in this case it it `Response`?

Reply:
Yes, to dynamically get the location of the assembly of the dll